src/managers/KeybindManager.cpp

```diff
@@ -744,7 +745,7 @@ SDispatchResult CKeybindManager::handleKeybinds(const uint32_t modmask, const SP
             Debug::log(ERR, "Invalid handler in a keybind! (handler {} does not exist)", k->handler);
         } else {
             // call the dispatcher
-            Debug::log(LOG, "Keybind triggered, calling dispatcher ({}, {}, {})", modmask, key.keyName, key.keysym);
+            Debug::log(LOG, "Keybind triggered, calling dispatcher ({}, {}, {}, {})", modmask, key.keyName, key.keysym, DISPATCHER->first);
```

Comment:
I don't think this change is needed

Reply:
why not? a message containing only numbers is dry and tasteless, some letters here make it understandable)

Reply:
`[LOG] Keybind triggered, calling dispatcher (64, , 107, cyclenext)`
is better, then just
`[LOG] Keybind triggered, calling dispatcher (64, , 107)`

Reply:
eh, fair enough, logs are disabled by default anyways.